tests/common/helpers/portchannel_to_vlan.py

```diff
@@ -0,0 +1,398 @@
+import pytest
+import ptf.testutils as testutils
+import collections
+import ipaddress
+import time
+import sys
+from netaddr import valid_ipv4
+
+from tests.common.helpers.assertions import pytest_require
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses    # noqa F401
+from tests.common.fixtures.duthost_utils import ports_list   # noqa F401
+from tests.common.fixtures.duthost_utils import utils_vlan_intfs_dict_orig          # noqa F401
+from tests.common.fixtures.duthost_utils import utils_vlan_intfs_dict_add
+from tests.common.helpers.backend_acl import apply_acl_rules, bind_acl_table
+from tests.generic_config_updater.gu_utils import create_checkpoint, rollback
```

Comment:
Can you also move the shared functions from `tests.generic_config_updater` to `tests/common'?

Reply:
Updated.